content/docs/practices/naming.md

```diff
@@ -26,6 +26,7 @@ A metric name...
    (exported by many client libraries)
  * <code><b>http</b>\_request\_duration\_seconds</code>
    (for all HTTP requests)
+* ...should not exceed 200 characters ( less than 100 is preferred )
```

Comment:
Thanks for this suggestion, I agree it makes sense to provide some guidance here.

Can you provide some examples for the long metric names you have come across? I assume users are encoding information in metric names, which should be set as labels instead. In that case, I'd suggest to change the wording accordingly to something like "...should not encode information, use labels for that" for example.